deepcell/applications/mesmer_test.py

```diff
@@ -222,6 +224,38 @@ def test_mesmer_app(self):
             self.assertEqual(x.shape[:-1], y.shape[:-1])
             self.assertEqual(y.shape[-1], 2)
 
+            # test that kwargs are passed through successfully
+            app._predict_segmentation = Mock()
+
+            # get defaults
+            _ = app.predict(x, compartment='whole-cell')
+            args = app._predict_segmentation.call_args[1]
+            default_cell_kwargs = args['postprocess_kwargs']['whole_cell_kwargs']
+            default_nuc_kwargs = args['postprocess_kwargs']['nuclear_kwargs']
+
+            # check that one arg is changed
+            maxima_threshold_cell = 0.2
+            radius_nuc = 4
+
+            _ = app.predict(x, compartment='whole-cell',
+                            postprocess_kwargs_whole_cell={'maxima_threshold':
+                                                           maxima_threshold_cell},
+                            postprocess_kwargs_nuclear={'radius': radius_nuc})
+
+            args = app._predict_segmentation.call_args[1]
+            whole_cell_kwargs = args['postprocess_kwargs']['whole_cell_kwargs']
+            assert whole_cell_kwargs['maxima_threshold'] == maxima_threshold_cell
+
+            nuclear_kwargs = args['postprocess_kwargs']['nuclear_kwargs']
+            assert nuclear_kwargs['radius'] == radius_nuc
+
+            # check that rest are unchanged
+            whole_cell_kwargs['maxima_threshold'] = 0.1
+            assert whole_cell_kwargs == default_cell_kwargs
+
+            nuclear_kwargs['radius'] = 2
+            assert nuclear_kwargs == default_nuc_kwargs
```

Comment:
Let's set this value based on the default_kwargs data, in case the defaults change in the future.